reconcile/test/test_utils_aws_api.py

```diff
@@ -11,7 +11,8 @@ def accounts():
             'name': 'some-account',
             'automationToken': {
                 'path': 'path',
-            }
+            },
+            'resourcesDefaultRegion': 'region'
```

Comment:
please change the value "region" to something different from the tf state bucket. how do you feel about creating a test that makes sure the session is really using the accounts region?

Reply:
added a test in https://github.com/app-sre/qontract-reconcile/pull/2165/commits/abc4d9681ff1ae2b60813713a867a81c404a6cb4